lilo

```diff
@@ -773,6 +775,22 @@ install_cups(){
   fi
 }
 #}}}
+#MICROCODE {{{
+ucode_cpu(){
+  print_title "MICROCODE - https://wiki.archlinux.org/index.php/Microcode"
+  print_info "MICROCODE is a computer hardware technique that imposes an interpreter between the CPU hardware and the programmer-visible instruction set architecture of the computer."  
```

Comment:
updated copypasta from 1st line of https://en.wikipedia.org/wiki/Microcode 

```
Microcode is a computer hardware technique that interposes a layer of organisation between the CPU hardware and the programmer-visible instruction set architecture of the computer.[1]
```
----

Remove current print_info and search. or use this article https://www.howtogeek.com/346627/what-is-firmware-or-microcode-and-how-can-i-update-my-hardware/  . First paragraph ->

## Firmware is a type of software that runs on a hardware device, performing low-level tasks. For example, everything from a television remote control to a computer hard drive to an aerial drone runs its own firmware. And microcode is basically firmware for your CPU.

----

<details>

<summary>the Wikipedia article is incorrect at some paragraphs. </summary>

<p>


Example 1->refer to first sentence of the wikipedia article. https://books.google.co.in/books?redir_esc=y&id=EjWV8J8CQEYC&q=microcode#v=snippet&q=microcode&f=false

Example 2:- https://www.logicalfallacies.info/
```
Some hardware vendors, especially IBM, use the term microcode as a synonym for firmware. In that way, all code within a device is termed microcode regardless of it being microcode or machine code; for example, hard disk drives are said to have their microcode updated, though they typically contain both microcode and firmware.[3]
```

</p>
</details>